app/src/androidTest/java/com/github/se/signify/ui/screens/challenge/ChallengeHistoryScreenTest.kt

```diff
@@ -42,21 +39,17 @@ class ChallengeHistoryScreenTest {
     composeTestRule.onNodeWithTag("TopBar").assertIsDisplayed()
     composeTestRule.onNodeWithTag("BackButton").assertIsDisplayed()
 
-    // Verify friends challenges achieved section is displayed with counts
-    composeTestRule.onNodeWithTag("FriendsChallengesRow").assertIsDisplayed()
-    composeTestRule.onNodeWithTag("FriendsChallengesText").assertIsDisplayed()
-    composeTestRule
-        .onNodeWithTag("FriendsChallengesText")
-        .assertTextEquals("Number of friends challenges achieved")
-    composeTestRule.onNodeWithTag("FriendsChallengesCountBox").assertIsDisplayed()
-
-    // Verify challenges created section is displayed with counts
-    composeTestRule.onNodeWithTag("ChallengesCreatedRow").assertIsDisplayed()
-    composeTestRule.onNodeWithTag("ChallengesCreatedText").assertIsDisplayed()
-    composeTestRule
-        .onNodeWithTag("ChallengesCreatedText")
-        .assertTextEquals("Number of challenges created")
-    composeTestRule.onNodeWithTag("ChallengesCreatedCountBox").assertIsDisplayed()
+    // Verify challenges section is displayed
+    composeTestRule.onNodeWithTag("ChallengesColumn").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("ChallengesRow").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("ChallengesText").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("ChallengesText").assertTextEquals("Number of challenges :")
+    composeTestRule.onNodeWithTag("Completed").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("Completed").assertTextEquals("Completed")
+    composeTestRule.onNodeWithTag("Created").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("Created").assertTextEquals("Created")
+    composeTestRule.onNodeWithTag("Won").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("Won").assertTextEquals("Won")
```

Comment:
I don't know how I feel about hardcoded `assertTextEquals()` in tests. It feels too restrictive.

Reply:
As these element are only implemented one time and never change, I don't think it is a problem.